sys/senml/senml.c

```diff
@@ -0,0 +1,467 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ * Copyright (C) 2016 Lennart Dührsen <lennart.duehrsen@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @author Lennart Dührsen <lennart.duehrsen@fu-berlin.de>
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <stdio.h>
+
+#include "senml.h"
+#include "jsmn.h"
+#include "cbor.h"
```

Comment:
nothing from this file is currently used in the implementation, right? So just drop the include for now.